pkg/adapter/info.go

```diff
@@ -23,6 +23,9 @@ import (
 type BuilderInfo struct {
 	// Name returns the official name of the adapter.
 	Name string
+	// ShortName is used to register this type with the config system.
```

Comment:
we have been using the full go package path as the name, which is useful for a developer to find code. These names are already unique. All this information is recorded in the CRD. one can derive short name from the full name. 
If we only require shortname, this info is lost.